includes/misc-functions.php

```diff
@@ -2288,3 +2288,58 @@ function give_get_safe_asset_url( $url ) {
 	 */
 	return apply_filters( 'give_get_safe_asset_url', $url );
 }
+
+/**
+ * Returns if you are on the new|edit post|page.
+ *
+ * string $post_action Check specifically for new or edit screen.
+ *
+ * @since 2.2.3
+ *
+ * @return boolean
+ */
+function give_is_editor_screen( $post_action = null ) {
+
+	global $pagenow;
+
+	// Make sure we are on the backend.
+	if ( ! is_admin() ) {
+		return false;
+	}
+
+	if ( 'edit' === $post_action ) {
+		return in_array( $pagenow, array( 'post.php' ) );
+	} elseif ( 'new' === $post_action ) {
+		return in_array( $pagenow, array( 'post-new.php' ) );
+	} else {
+		return in_array( $pagenow, array( 'post.php', 'post-new.php' ) );
+	}
+}
+
+/**
+ * Returns if the editor page is a Gutenberg editor.
+ *
+ * @since 2.2.3
+ *
+ * @return boolean
+ */
+function give_is_gutenberg_editor() {
+
+	// Check if Gutenberg is active.
+	$is_gutenberg_active = is_plugin_active( 'gutenberg/gutenberg.php' );
```

Comment:
@Sidsector9 what happens when Gutenberg is merged into WP Core? Will this return properly since it will no longer be a plugin? Please take into account when it is merged into core. Is there a codex option that's necessary?

Reply:
@Sidsector9 can you simply check the version of WP here for 5.0 plus so we don't have to go back and refactor?

Reply:
@DevinWalker I have added check for WPv5 and also for active Gutenberg plugin if WP version is < 5.